backend/src/test/resources/graphql-test/edit-queue-item-multiplex-result-mutation.graphql

```diff
@@ -0,0 +1,14 @@
+mutation EditQueueItemMultiplexResult($id: ID!, $deviceId: String, $results: [MultiplexResultInput], $dateTested: DateTime) {
```

Comment:
This is not a new file, it was just moved to `backend/src/test/resources/graphql-test` dir, git is not showing that correctly.